.github/workflows/manage-github-issue-for-outdated-pods.yml

```diff
@@ -0,0 +1,34 @@
+name: Check outdate pod and create issue
+
+on: 
+  schedule:
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+
+jobs:
+  check-outdated-pods:
+    runs-on: macos-latest
+    name: Check outdate pods and create issue if it doesn't exist
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check outdated pods and create issue
+        id: check-outdated-pods-and-create-issue
+        uses: rudderlabs/github-action-updated-pods-notifier@main
+        with:
+          outdated-pod-names: "Amplitude"
+          directory: "Example"
+          title: "fix: update Amplitude SDK to the latest version"
+          assignee: "desusai7"
+          labels: "outdatedPod"
+          color: "FBCA04"
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - name: Get the github issue url
+        if: steps.check-outdated-pods-and-create-issue.outputs.issue-url != ''
+        run: echo "The Github issue url is ${{ steps.check-outdated-pods-and-create-issue.outputs.issue-url }}"
```

Comment:
The `Issue URL` is returned as output from our shareable GitHub action and I've included it in this action only for display and it is totally optional. 
If you think it's unnecessary, I can remove it, let me know.

Reply:
you can print in the common action the Issue URL.

common actions can have these steps

Reply:
It is already being printed using common action.
I thought it would be better to have an extra step to `only display` the Issue URL. Hence, I made it as a separate step.
LMK, if you think, it would be better to remove that extra step to display the Issue URL?

Reply:
sure lets keep them then